### DISCUSSION TITLE
fix permissions did not correct attached issue

### DISCUSSION THREAD

**Discussion by trsbox** [Chia-Network/chia-blockchain#8567] [Support]
[message.txt](https://github.com/Chia-Network/chia-blockchain/files/7212393/message.txt)

**Comment by loppefaaret** [comment]
run `chia init --fix-permissions` from inside a (venv) after having done the `. ./activate` to enter the (venv) - that should fix the permissions, if the filesystem supports linux permissions

**Comment by BrandtH22** [comment]
@trsbox , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
